frontends/web/src/routes/lightning/receive/receive.tsx

```diff
@@ -40,13 +40,17 @@ import { unsubscribe } from '../../../utils/subscriptions';
 import { Spinner } from '../../../components/spinner/Spinner';
 import { Checked, Copy, EditActive } from '../../../components/icon';
 import styles from './receive.module.css';
+import { FiatConversion } from '../../../components/rates/rates';
+import { getBtcSatsAmount } from '../../../api/coins';
+import { IAmount } from '../../../api/account';
 
 type TStep = 'select-amount' | 'wait' | 'invoice' | 'success';
 
 export function Receive() {
   const { t } = useTranslation();
-  const [amountSats, setAmountSats] = useState<number>(0);
-  const [amountSatsText, setAmountSatsText] = useState<string>('');
```

Comment:
In send I refactored this, as we don't need 2 states holding the same data once a number and once as string...

We should refactor this at some point here as well.

Reply:
Done